modules/cluster-logging-release-notes-5.3.z.adoc

```diff
@@ -1,5 +1,48 @@
 //Z-stream Release Notes by Version
 
+[id="cluster-logging-release-notes-5-3-4"]
+= OpenShift Logging 5.3.4
+This release includes link:https://access.redhat.com/errata/RHBA-2022:0411[RHBA-2022:0411 OpenShift Logging Bug Fix Release 5.3.4]
+
+[id="openshift-logging-5-3-4-bug-fixes"]
+== Bug fixes
+* Before this update, changes to the metrics dashboards had not yet been deployed because the cluster-logging-operator did not correctly compare existing and desired configmaps containing the dashboard.  This update fixes the logic by adding unique hash value to the object labels. (link:https://issues.redhat.com/browse/LOG-2066[LOG-2066])
```

Comment:
```suggestion
* Before this update, changes to the metrics dashboards had not yet been deployed because the `cluster-logging-operator` did not correctly compare existing and desired config maps that contained the dashboard.  This update fixes the logic by adding a unique hash value to the object labels. (link:https://issues.redhat.com/browse/LOG-2066[LOG-2066])
```